pos_order_to_sale_order_commitment_date/models/pos_config.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Binhex - Adasat Torres de León.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    iface_sale_order_allow_commitment_date = fields.Boolean()
```

Comment:
I guess that if we install the module, we want the feature to be enabled. Don't you think ? 
```suggestion
    iface_sale_order_allow_commitment_date = fields.Boolean(default=True)
```